src/components/bookItem.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] @Reggeditt it's not good practice to disable linter checks in your file, kindly remove this line and solve the linter errors instead of disabling the check.

Reply:
thanks1